swift-blackjack-viewsTests/PlayerSpec.swift

```diff
@@ -0,0 +1,283 @@
+//  PlayerSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class PlayerSpec: QuickSpec {
+    override func spec() {
+        describe("Player") {
+            
+            var player = Player(name: "Player")
+            
+            let aceOfSpades : Card = Card(suit: "♠︎", rank: "A")
+            let aceOfHearts : Card = Card(suit: "♥︎", rank: "A")
+            let queenOfHearts : Card = Card(suit: "♥︎", rank: "Q")
+            let tenOfDiamonds : Card = Card(suit: "♦︎", rank: "10")
+            let twoOfClubs : Card = Card(suit: "♣︎", rank: "2")
+
+            beforeEach({ () -> () in
```

Comment:
cool. it was autocomplete syntax that i guess is implicit if i don't fill it in with anything?